source/Playnite.DesktopApp/ViewModels/SettingsViewModel.cs

```diff
@@ -233,6 +233,71 @@ public RelayCommand<object> WindowClosingCommand
             });
         }
 
+        public RelayCommand<object> AddSortingNameRemovedArticle
+        {
+            get => new RelayCommand<object>(_ =>
+            {
+                var res = dialogs.SelectString(
+                    resources.GetString("LOCEnterName"),
+                    resources.GetString("LOCAddNewItem"),
+                    "");
+                if (res.Result && !res.SelectedString.IsNullOrEmpty())
+                {
+                    if (settings.GameSortingNameRemovedArticles.Any(a => a.Equals(res.SelectedString, StringComparison.InvariantCultureIgnoreCase)))
+                    {
+                        dialogs.ShowErrorMessage(resources.GetString("LOCItemAlreadyExists"), "");
+                    }
+                    else
+                    {
+                        settings.GameSortingNameRemovedArticles.Add(res.SelectedString);
+                        settings.GameSortingNameRemovedArticles = settings.GameSortingNameRemovedArticles.GetClone();

```

Comment:
I would just create backing property that implements observable collection and update source settings when settings are being saved. Like we already do that with `ImportExclusionList` list.